### DISCUSSION TITLE
Hospedagem para clientes (freelancer)

### DISCUSSION THREAD

**Discussion by Matheus-Barbosa01** [frontendbr/forum#2492] [Discussão]
Pessoal estou começado agora e vou começar a fazer sites simples, porém não sei em qual local hospedar o site (Obs, vou usar: **HTML, CSS e JavaScript**).  Ouvir sobre o netlify e hostinger, qual é o melhor para isso ? se o cliente quiser um domino, qual devo usar ? se tiver outros locais que vocês recomenda podefalar também, eu agradeço.

**Comment by felquis** [comment]
Usei vários no decorrer dos anos, hoje em dia pra mim não tem um que seja o melhor.

**O melhor é o que funciona pra você de acordo com suas habilidades**

Para mim, o CloudFlare tem funcionado muito bem e recomendo dar uma olhada tentar publicar algo através dele pra testar.

**Comment by rafgam10** [comment]
Estou na mesma dúvida, quero começar a fazer feelar, mas quero sabe pelo menos deixar o site ou software do cliente para hospedar free entende.
